### ISSUE TITLE
Support of pyqt6

### ISSUE THREAD

**Issue by hakdogpinas** [pyqtconsole/pyqtconsole#71]
Hey, any update if pyqtconsole will support pyqt6? I need console in my project which will be embed to pyqt6 ui

**Comment by stefanseefeld** [comment]
Hi, I notice a few "recent" updates to the code that haven't made it into a release yet. I'm in the middle of moving a project from PyQt5 to PyQt6, and pyqtconsole is part of my software stack. Is there anything I can do to help move the project along ?
Thanks !

**Comment by marcus-oscarsson** [comment]
HI all, I'm not actively using pyqtconosle any longer, and sort of just trying to address the small things that I can. We are more than happy to accept PR's or adding Qt6 support.

**Comment by marcus-oscarsson** [comment]
Perfect !, thanks alot @RobertoRoos

**Comment by stefanseefeld** [comment]
@RobertoRoos what version did you run your test with ? The last release or current master ? The last release doesn't work for me (notably because it's using QRegExp, which no longer exists in PyQt6).

@marcus-oscarsson , please consider tagging a new release, assuming master is in a stable state.

Many thanks !

**Comment by RobertoRoos** [comment]
> @RobertoRoos what version did you run your test with ? The last release or current master ? The last release doesn't work for me (notably because it's using QRegExp, which no longer exists in PyQt6).

Yeah, with `master`. With the latest release I indeed get the same errors.

A new release then seems obvious, I see now reason to make it.
